docs/src/catalyst_functionality/programmatic_CRN_construction.md

```diff
@@ -61,6 +61,9 @@ system to be the same as the name of the variable storing the system.
 Alternatively, one can use the `name = :repressilator` keyword argument to the
 `ReactionSystem` constructor.
 
+!!! warn
+       All `ReactionSystem`s created programmatically (i.e. by calling `ReactionSystem` with some input, rather than using `@reaction_network`) are created as *incomplete*. To simulate them, they must first be made *complete*. This can be done using the `complete` function, i.e. by calling `repressilator = complete(repressilator)`. An expanded description on *completeness* can be found [here](@ref completeness_note).
+
```

Comment:
As mentioned earlier, this PR does not make the most extensive rewrite of the docs. But I do add minimally what would be required for someone to be able to run things given the new changes.